src/api/app/helpers/webui/buildstatus_helper.rb

```diff
@@ -0,0 +1,63 @@
+module Webui::BuildstatusHelper
+  # rubocop:disable Style/AlignHash
```

Comment:
@mschnitzer Between unresolvable: and it's value you have 2 whitespaces. When you remove that one, it passes in rubcop.